com.sap.cloud.lm.sl.cf.web/src/test/java/com/sap/cloud/lm/sl/cf/web/resources/ConfigurationEntriesResourceTest.java

```diff
@@ -281,9 +277,6 @@ public PostRequestTest(PostRequestTestInput input, Expectation expectation) {
         protected void test() throws Exception {
 
             TESTER.test(() -> {
```

Comment:
test(() -> new RestResponse(resource.createConfigurationEntry(input.getEntryXml()))), expectation)

Reply:
There are a lot of similar stuff in all of our tests. This refactoring better be done in another PR as this one got huge already, and it should fix all occurances, not just this one. It is also not the topic for the listed BIs. It is a '4th' side refactoring I'd do if I was to do it.

Reply:
Agreed